### DISCUSSION TITLE
Add example with async_graphql

### DISCUSSION THREAD

**Discussion by IgnisDa** [apalis-dev/apalis#87] [Q&A] [answered]
It would be really helpful if an example with [async graphql](async-graphql.github.io/) was present. I am using axum and async graphql and I can not access the `storage` in the `gql_ctx`. I can not figure out how to push a new job.

**Comment by geofmureithi** [comment, marked as answer]
You should be able to use `.data`
https://github.com/async-graphql/examples/blob/7eee23ad4e005ac180df8111e6940cebb7c1f52f/axum/starwars/src/main.rs#L25
